contents/swift/transition-to-swift.md

```diff
@@ -0,0 +1,340 @@
+# Swift
+
+Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md)
+
+# Overview
+
+Introduced in 2014, Swift is a fast, modern, safe and interactive programming language. If you have ever thought of building an app for your MacBook or iPhone (or any other Apple Product), Swift is the language for you. With its syntactic sugar, it is much more readable compared to it's "predecessor" Objective-C(ObjC). Together with its Playground feature, you can run your code easily, making Swift easy to learn.
```

Comment:
You may want to put links for these terms like `syntactic sugar` `Objective-C` `Playground feature`